### DISCUSSION TITLE
[bug]: Huge number of inactive channels after updating to lnd v0.15.2-beta

### DISCUSSION THREAD

**Discussion by peername** [lightningnetwork/lnd#7038] [Troubleshooting]
### Background

Most of my channels are still inactive hours after the update to lnd v0.15.2-beta

This is repeated over and over in the logs:

> 2022-10-10 21:09:40.796 [WRN] HSWC: ChannelLink(3abecbc8d5b5dd84d23c88fe4562e6ba16a9d32b9e4e8cd857f5c5db0c022c7c:0): insufficient bandwidth to route htlc: 12503837 mSAT is larger than 2122 mSAT
> 2022-10-10 21:09:40.796 [ERR] HSWC: insufficient bandwidth to route htlc
> 2022-10-10 21:09:40.796 [ERR] HSWC: Unhandled error while reforwarding htlc settle/fail over htlcswitch: insufficient bandwidth to route htlc

### Your environment

* version of `lnd`: v0.15.2-beta
* which operating system (`uname -a` on *Nix): Ubuntu Server 20.04
* version of `btcd`, `bitcoind`, or other backend: bitcoind 23

**Comment by sputn1ck** [comment]
I think the number of inactive channnels could in the expected range 12 hours after release of the update, probably not everyone is quick in updating. For me roughly 50% are still inactive.

I notice the same error messages repeating as you, gonna check back later what the cause may be.

**Comment by Roasbeef** [comment]
That's a log where someone is trying to route over a channel but it has insufficient balance to carry it. Which is sort of expected if someone is trying to probe.

**Comment by Roasbeef** [comment]
Also if your node on clearnet, tor, or hybrid?

**Comment by peername** [comment]
hybrid

**Comment by sputn1ck** [comment]
For me about 85% of channels are active again, whats your status?

**Comment by sputn1ck** [comment]
Moving this issue to a discussion as it does not appear to be a bug

**Reply by gogomickey** [reply]
Any updates on this?  I have same issue.  I'm operating a Node over tor.

**Reply by Roasbeef** [reply]
Is the node Tor only? So it must be the one to connect outbound to non-tor peers? Are you able to try and manually reconnect the peers, does that work?
